docpad.coffee

```diff
@@ -6,7 +6,7 @@
 docpadConfig = {
 
   templateData:
-    version: '1.9.1'
+    version: '3.2.2'
```

Comment:
this is the jsforce version, will be used in URL templates for loading it from jsdeliver.